po/ar.po

```diff
@@ -976,109 +982,109 @@ msgstr "اليوم الأول: صباحا"
 
 #: src/SUMMARY.md src/bare-metal/no_std.md
 msgid "`no_std`"
-msgstr ""
+msgstr "`no_std`"
 
 #: src/SUMMARY.md
 msgid "A Minimal Example"
-msgstr ""
+msgstr "مثال بسيط"
 
 #: src/SUMMARY.md src/bare-metal/no_std.md src/bare-metal/alloc.md
 msgid "`alloc`"
-msgstr ""
+msgstr "`alloc`"
 
 #: src/SUMMARY.md src/bare-metal/microcontrollers.md
 msgid "Microcontrollers"
-msgstr ""
+msgstr "المتحكمات الدقيقة (Microcontrollers)"
 
 #: src/SUMMARY.md src/bare-metal/microcontrollers/mmio.md
 msgid "Raw MMIO"
-msgstr ""
+msgstr "MMIO الخام (Raw MMIO)"
 
 #: src/SUMMARY.md
 msgid "PACs"
-msgstr ""
+msgstr "حزم الوصول المحيطي (PACs)"
 
 #: src/SUMMARY.md
 msgid "HAL Crates"
-msgstr ""
+msgstr "حزم تجريد الأجهزة (HAL Crates)"
```

Comment:
I think the translation for "Layer" is missing

Reply:
you are right

Reply:
```suggestion
msgstr "حزم طبقة تجريد الأجهزة (HAL Crates)"
```